packages/web/src/components/gcds-link/stories/gcds-link.stories.js

```diff
@@ -0,0 +1,211 @@
+export default {
+  title: 'Components/Link',
+
+  parameters: {
+    actions: {
+      argTypesRegex: '^on.*',
+      handles: ['click', 'change', 'focus', 'blur'],
+    },
+  },
+
+  argTypes: {
+    // Props
+    size: {
+      control: { type: 'radio' },
+      options: ['regular', 'small', 'inherit'],
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'regular' },
+      },
+    },
+    display: {
+      control: 'select',
+      options: ['inline', 'inline-block', 'block'],
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'inline' },
+      },
+    },
+    href: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    rel: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    target: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '_self' },
+      },
+    },
+    external: {
+      control: 'boolean',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: false },
+      },
+    },
+    download: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    type: {
+      control: { type: 'text' },
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+
+    // Slots
+    default: {
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'Slots | Fentes',
+      },
+    },
+  },
+};
+
+const Template = args =>
+  `
+This is an example of
+<!-- Web component code (Angular, Vue) -->
+<gcds-link ${args.display ? `display="${args.display}"` : null} ${
+    args.href ? `href="${args.href}"` : null
+  } ${args.rel ? `rel="${args.rel}"` : null} ${
+    args.target ? `target="${args.target}"` : null
+  } ${args.size != 'regular' && args.size ? `size="${args.size}"` : null} ${
+    args.external ? `external` : null
+  } ${args.download ? `download="${args.download}"` : null} ${
+    args.type ? `type="${args.type}"` : null
+  }>${args.default}</gcds-link>
+
+<!-- React code -->
+<GcdsLink ${args.display ? `display="${args.display}"` : null} ${
```

Comment:
Same comment about the display prop, can we change that to the following to only have it displayed if it's not set to its default value?
```${args.display != 'inline' ? `display="${args.display}"` : null}```